src/content/blog/2024-06-20-simple-open-payments-guide.md

```diff
@@ -0,0 +1,102 @@
+
+For most people, accessing an Open Payments-enabled account depends on their financial institutions adopting the standard. As more banks, digital wallets, and mobile money providers incorporate Open Payments, the benefits will become more widely accessible.
+
+## What Are the Possibilities for Tomorrow?
+
+A world with direct access to accounts through the Open Payments standard fosters innovation and inclusion by reducing barriers to entry for developers, who can create direct payment solutions more quickly and at a lower cost without the need for custom integrations. This democratization of development enables smaller companies and startups to compete and innovate.
+
+Users are empowered to give applications direct access to their accounts. They can give access to who they want, to do what they want within set limits. Users also benefit from seamless transactions, transparency with regards to what fees they incur, and enhanced control over their financial data, improving trust and security. Financial inclusion is significantly increased by enabling access for those without traditional bank accounts, as well as cutting down on the costs of making payments by removing unnecessary intermediaries, and reducing the development effort of payment integrations.
+
+## Try Open Payments Yourself
+
+If you're curious about how the Open Payments standard works in practice, you can explore it by creating an account on the [Interledger Test Wallet](https://rafiki.money/). Upon account creation you'll be given a wallet address. This enables you to get hands on experience of simulating transactions using Open Payments. Once you're set up with your account, and have given yourself a generous amount of fictional money, you can embark on a shopping spree at the [Interledger Boutique](https://rafiki.boutique) and buy some delightful products ranging from luck to kindness.
+
+## TLDR
+
+Standardization enhances interoperability by reducing the development effort required for each integration. This alone brings down costs while providing a scalable and efficient solution for handling digital payments. When interoperability is easy, there is no need for unnecessary middlemen and the fees they add to the cost of transactions.
+
+With Open Payments, users are empowered to give applications direct access to their accounts, deciding who can access their money, how much, and how often. Users only share public information (in the form of wallet addresses) with applications when making payments, keeping their sensitive financial data private.
+
+Applications can handle payments without needing to be registered financial service providers or navigating the risks involved in handling sensitive information like credit card numbers.
```

Comment:
Should we move that to the top to be an actual TLDR?

Reply:
What if we change the name to "Key Takeaways" instead. 

My reluctance with moving it to the top is that I want to create a gentle path through the post that encourages an incremental introduction to the concepts. I worry it's kind off putting to put the summary at the top. People who are less technical might feel like this article is pitched over their heads. But by the time they reach the bottom they should understand all of this and it just re-captures the main points to give a simple summary of what OP is.

What do you think?